cmd/trust-manager/app/options/options.go

```diff
@@ -82,9 +121,20 @@ func (o *Options) Prepare(cmd *cobra.Command) *Options {
 // Complete will populate the remaining Options from the CLI flags. Must be run
 // before consuming Options.
 func (o *Options) Complete() error {
-	klog.InitFlags(nil)
-	log := klogr.New()
-	flag.Set("v", o.logLevel)
```

Comment:
comment: Unrelated to this PR but I'd never actually stopped to consider how bonkers it is that `flag.Set("v", o.logLevel)` is how a verbosity level _should_ be set according to klogr!

Reply:
Agree, but that's history after this PR. 😉